test/kongintegration/translator_golden_tests_outputs_test.go

```diff
@@ -17,28 +19,23 @@ import (
 
 	"github.com/kong/kubernetes-ingress-controller/v3/internal/dataplane/sendconfig"
 	"github.com/kong/kubernetes-ingress-controller/v3/test/internal/helpers"
+	"github.com/kong/kubernetes-ingress-controller/v3/test/internal/helpers/konnect"
 	"github.com/kong/kubernetes-ingress-controller/v3/test/kongintegration/containers"
 )
 
-// TestGoldenTestsOutputs ensures that the translators's golden tests outputs are accepted by Kong.
+const (
+	timeout = 5 * time.Second
+	tick    = 100 * time.Millisecond
+)
```

Comment:
Ideally we'd put this into a dedicated file, something like `consts.go` where all tests from this package could use it.